prometheus_hardware_exporter/collectors/lshw.py

```diff
@@ -0,0 +1,32 @@
+"""Lshw metrics collector."""
+import json
+from logging import getLogger
+from typing import Dict, List
+
+from ..utils import Command
+
+logger = getLogger(__name__)
+
+
+class LSHW(Command):
+    """Command line tool for lshw."""
+
+    prefix = ""
+    command = "lshw"
+
+    def get_powers(self) -> List[Dict[str, str]]:
+        """Get power class data."""
+        result = self("-c power -json")
```

Comment:
why did you choose to use lshw instead of dmidecode?
On a server I am testing I get similar but not identical results:
```
$ sudo dmidecode -t 39 | grep 'Power Capacity'
	Max Power Capacity: 550 W
	Max Power Capacity: 550 W

$ sudo lshw -c power | grep capacity
       capacity: 550mWh
       capacity: 550mWh
```

(obviously Wh != W)

Reply:
$$P(W) = E(Wh) / t(h)$$

Yes, using `W` will be more correct. Let's change to dmidecode.

Reply:
Updated